### PR TITLE
Mention that changing r_{s,t} definition is troublesome

### PR DESCRIPTION
**Pull request by abelsiqueira**
Regarding #30. See discussion before merging.

### CODE REVIEW

File: reuse.tex [hunk 1]
```diff
@@ -7,9 +7,12 @@
 which creates the expected figure.
 The user shall have little difficult creating his own backend, specially if he
 uses one of our own as a starting point.
+However, if the user wants to change the definition of the profile data (as in
+to try to implement the data profile), we would have modify the {\tt Pdata}
```
Comment (rgaiacs) [inline reuse.tex]: I suggest to write

> we would have to modify the {\tt set_percent_problems_solved_by_time} method of {\tt Pdata}

**Comment by abelsiqueira** [comment]
Updated

**Comment by abelsiqueira** [comment]
Corrected

**Comment by abelsiqueira** [comment]
Thanks, removed.

**Pull request merged**